modules/doc/content/getting_started/installation/inl_hpc_install_moose.md

```diff
@@ -0,0 +1,138 @@
+# INL HPC Cluster
+
+This page aims at simplifying the installation process on INL computing clusters.
+The [general cluster instructions](hpc_install_moose.md) may also be used, but these
+instructions will save some steps. There are multiple options for installing / using
+MOOSE and MOOSE applications.
+
+!alert note
+Falcon is planned to be decommissioned this January 2022.
+
+!alert warning
+Do not mix installation workflows except when specified here.
+
+## Option 1: Loading an application directly (easy, users only)
+
+You may load pre-built binaries for MOOSE
+and the applications you have access to by running: `module load use.moose moose-apps <app_name>`.
+Pre-built executables are then added in the `PATH` environment variable, and can be called
+directly in any folder with: `app_name-opt -i <input_file>`.
+
+## Option 2: Modules + source-based installation (medium, for developers)
+
+### Step 1: load some compilers
+
+In order to compile MOOSE and its applications, we first need to load compilers & compiling tools.
+The following modules are recommended for each of INL's clusters:
+
+- Falcon: `module load use.moose MVAPICH2 CMake`
```

Comment:
Does use.moose already include PETSc?

Reply:
use.moose opens the door to load other modules. Think of it like changing into a directory and performing a new directory listing.

So while use.moose doesn't load a PETSc module, it allows for one to be loaded.

Reply:
nope, or at least not on sawtooth

Reply:
also if you load petsc without it, you will get an intel compiler one (on at least one of the machines)